mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
```

Comment:
It's probably worth including the [`python_version`](https://mypy.readthedocs.io/en/stable/config_file.html#confval-python_version) to ensure that MyPy is applying the proper typing rules for the target version that this will be running under, rather than just what happens to be running on the machine that executes the mypy command.

Reply:
Sure, done.